### PR TITLE
added rosbag_to_image code

### PR DESCRIPTION
**Pull request by sophiazhang14**
I added a node that turns rosbags to png.

### CODE REVIEW

File: all_seaing_utility/all_seaing_utility/rosbag_to_image.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import cv2
+from cv_bridge import CvBridge
+import os
+import rosbag2_py
+from rclpy.serialization import deserialize_message
+from sensor_msgs.msg import Image
+import rclpy
+
+def main():
+    rclpy.init()
```
Comment (toyat522) [inline all_seaing_utility/all_seaing_utility/rosbag_to_image.py:15]: I think for this script you won't need rclpy.init() or shutdown(). Maybe I'm wrong though, so you should check.

File: all_seaing_utility/all_seaing_utility/rosbag_to_image.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    reader.open(storage_options, converter_options)
+
+    bridge = CvBridge()
+    count = 0
+
+    print(f"Extracting images from {bag_file} on topic {image_topic} into {output_dir}")
+
+    choose_framerate = 5
+    current_rate = 579//38.314721344 #15
```
Comment (toyat522) [inline all_seaing_utility/all_seaing_utility/rosbag_to_image.py:42]: Wondering how you got those numbers (was it from ros2 bag info?). I wonder if there is a way to calculate this value without hard-coding. If not, I'm also okay with a parameter like "sample_every" (e.g. sample_every 2 can get every 2 messages, just like how you are getting every framerate messages right now) and add it to the "parser.add_argument" above. I think ideally the people who use this script don't need to edit this file ever, just needs to provide the arguments.

File: all_seaing_utility/all_seaing_utility/rosbag_to_image.py (hunk 1 above)
Comment (sophiazhang14) [reply all_seaing_utility/all_seaing_utility/rosbag_to_image.py:15]: Does anything bad happen if run init & shutdown when it's not needed?

Comment (toyat522) [reply all_seaing_utility/all_seaing_utility/rosbag_to_image.py:15]: Nothing "bad" would happen, but I don't think it is necessary for this script? It's just nice to not add unused code.

File: all_seaing_utility/all_seaing_utility/rosbag_to_image.py (hunk 2 above)
Comment (sophiazhang14) [reply all_seaing_utility/all_seaing_utility/rosbag_to_image.py:42]: I got the numbers from ros2 bag info. In the new commit, I added the argument for fps and it finds fps through the code instead.

**Pull request merged**